File: C#/Programming Basics with C#/13. Nested Loops - Exercise/02. Number Pyramid/NumberPyramid.cs
using System;

class NumberPyramid
{
    static void Main()
    {
        int n = int.Parse(Console.ReadLine());
        int counter = 0;

        for (int rows = 1; rows <= n; rows++)
        {
            for (int columns = 1; columns <= rows; columns++)
            {

                ++counter;
                Console.Write(counter + " ");
                if (counter == n)
                {
                    break;
                }
            }

            Console.WriteLine();

            if (counter == n)
            {
                break;
            }
        }
    }
}

File: C++/Programming Basics with C++/13. Exercise - Nested loops/simple_task/NumberPyramid.cpp
#include <iostream>

using namespace std;

int main() {
    int n;
    cin >> n;
    int counter = 0;

    for (int rows = 1; rows <= n; rows++) {
        for (int columns = 1; columns <= rows; columns++) {

            ++counter;
            cout << counter << " ";
            if (counter == n) {
                break;
            }
        }

        cout << endl;

        if (counter == n) {
            break;
        }
    }

    return 0;
}
